docs/_includes/phone-input.html

```diff
@@ -0,0 +1,1176 @@
+              <div class="listbox-button__option" role="option">
+                <span class="listbox-button__value">
+                  <svg class="flag flag--sh" focusable="false" height="18" width="24" aria-hidden="true"
+                    data-country-code="290">
+                    <use href="#flag-sh"></use>
+                  </svg>
+                  <span>Saint Helena, Ascension and Tristan da Cunha</span>
+                  <span>(+290)</span>
+                </span>
+                <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+                  <use href="#icon-tick-16"></use>
+                </svg>
+              </div>
+              <div class="listbox-button__option" role="option">
+                <span class="listbox-button__value">
+                  <svg class="flag flag--gb" focusable="false" height="18" width="24" aria-hidden="true"
+                    data-country-code="44">
+                    <use href="#flag-gb"></use>
+                  </svg>
+                  <span>United Kingdom</span>
+                  <span>(+44)</span>
+                </span>
+                <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+                  <use href="#icon-tick-16"></use>
+                </svg>
+              </div>
+              <div class="listbox-button__option" role="option" aria-selected="true">
+                <span class="listbox-button__value">
+                  <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true"
+                    data-country-code="1">
+                    <use href="#flag-us"></use>
+                  </svg>
+                  <span>United States of America</span>
+                  <span>(+1)</span>
+                </span>
+                <svg class="icon icon--tick-16" focusable="false" height="10" width="14" aria-hidden="true">
+                  <use href="#icon-tick-16"></use>
+                </svg>
+              </div>
+            </div>
+          </div>
+        </span>
+        <span class="textbox">
+          <span id="phone-prefix-2">+1</span>
+          <input class="textbox__control" type="tel" aria-label="area code + phone number" autocomplete="tel-national"
+            aria-describedby="phone-prefix-2" />
+        </span>
+      </span>
+    </div>
+  </div>
+
+  {% highlight html %}
+  <span class="phone-input phone-input--large">
+    <span class="listbox-button listbox-button--form">
+      <button class="btn btn--form btn--split-start btn--large btn--large-fixed-height" aria-expanded="false"
+        aria-haspopup="listbox" type="button" data-listbox-button-prefix="Country: ">
+        <span class="btn__cell">
+          <span class="btn__text">
+            <svg class="flag flag--us" focusable="false" height="18" width="24" aria-hidden="true" data-country-code="1"
+              aria-label="Country: United States of America (+1)">
```

Comment:
@saiponnada I notice this example code has the aria-label on the svg instead of the button.